src/pocketmine/event/Listener.php

```diff
@@ -23,6 +23,32 @@
 
 namespace pocketmine\event;
 
+/**
+ * Classes implementing this interface can be registered to receive called Events when something happens.
+ *
+ * Event handlers do not have to have any particular name - they are detected using reflection.
+ *
+ * A function in a Listener class must meet the following criteria to be registered as an event listener:
+ * - MUST be public
+ * - MUST NOT be static
+ * - MUST accept EXACTLY ONE class parameter which:
+ *   - MUST be a VALID class extending Event
+ *   - MUST NOT be abstract, UNLESS it has an @allowHandle annotation
+ *
+ * Functions which meet the criteria can have the following annotations in their doc comments:
```

Comment:
Can be better phrased as "listener functions". Otherwise this sounds like functions not meeting the criteria can't have the annotations due to restrictions.

Reply:
I don't see how that would make it any clearer...